Psorcast/PsorcastValidation/BellwetherStepViewController.swift

```diff
@@ -188,30 +201,48 @@ open class BellwetherStepViewController: RSDStepViewController, BellwetherImageV
             !(backgroundTheme is RSDAnimatedImageThemeElement) {
             
             if let assetLoader = backgroundTheme as? RSDAssetImageThemeElement {
-                self.backgroundImageBack = assetLoader.embeddedImage()
-                if !self.initialResultFront {
-                    self.backgroundImageView?.image = assetLoader.embeddedImage()
-                }
+                self.backgroundImageViewBack.image = assetLoader.embeddedImage()
             } else if let fetchLoader = backgroundTheme as? RSDFetchableImageThemeElement {
                 fetchLoader.fetchImage(for: frontSize, callback: { (_, img) in
-                    self.backgroundImageBack = img
-                    if !self.initialResultFront {
-                        self.backgroundImageView?.image = img
-                    }
+                    self.backgroundImageViewBack.image = img
                 })
             }
         }
     }
     
     override open func showLearnMore() {
-        if self.bellwetherImageView.currentRegion == .front {
+        if !self.backgroundContainerFront.isHidden {
             self.learnMoreButton?.setTitle(Localization.localizedString("VIEW_MY_FRONT_BUTTON"), for: .normal)
-            self.bellwetherImageView.currentRegion = .back
-            self.backgroundImageView?.image = self.backgroundImageBack
-        } else if self.bellwetherImageView.currentRegion == .back {
+            self.showBack(animate: true)
+        } else {
             self.learnMoreButton?.setTitle(Localization.localizedString("VIEW_MY_BACK_BUTTON"), for: .normal)
-            self.bellwetherImageView.currentRegion = .front
-            self.backgroundImageView?.image = self.backgroundImageFront
+            self.showFront(animate: true)
+        }
+    }
+    
+    func showFront(animate: Bool) {
+        let showFrontFunc: ((Bool) -> Void) = { (success) in
+            self.backgroundContainerFront.isHidden = false
+            self.backgroundContainerBack.isHidden = true
+        }
+        
+        if animate {
+            UIView.transition(from: self.backgroundContainerBack, to: self.backgroundContainerFront, duration: 1, options: [.transitionFlipFromRight, .showHideTransitionViews], completion: showFrontFunc)
+        } else {
+            showFrontFunc(true)
+        }
+    }
+    
+    func showBack(animate: Bool) {
+        let showBackFunc: ((Bool) -> Void) = { (success) in
+            self.backgroundContainerFront.isHidden = true
+            self.backgroundContainerBack.isHidden = false
+        }
+        
+        if animate {
+            UIView.transition(from: self.backgroundContainerFront, to: self.backgroundContainerBack, duration: 1, options: [.transitionFlipFromLeft, .showHideTransitionViews], completion: showBackFunc)
```

Comment:
Animation looks great, its just a little slow. After playing a bit, I suggest a change to 0.5 seconds. 
```suggestion
            UIView.transition(from: self.backgroundContainerFront, to: self.backgroundContainerBack, duration: 0.5, options: [.transitionFlipFromLeft, .showHideTransitionViews], completion: showBackFunc)
```